src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -261,6 +261,16 @@ public interface IDurableOrchestrationClient
         /// <returns>Returns a task which completes when the status has been fetched.</returns>
         Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId, bool showHistory = false, bool showHistoryOutput = false, bool showInput = true);
 
+        /// <summary>
+        /// Get the status of multiple instances.
+        /// </summary>
+        /// <param name="instanceID"> List of multiple instance ID that needed to check that status.</param>
+        /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
+        /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
+        /// <param name="showInput">If set, fetch and return the input for the orchestration instance.</param>
+        /// <returns>return a list of instances of orchestration status.</returns>
+        Task<IList<DurableOrchestrationStatus>> GetStatusAsync(IEnumerable<string> instanceID, bool showHistory = false, bool showHistoryOutput = false, bool showInput = true);
```

Comment:
Let's make the default value for `showInput` false. This is a better default for us to use for new APIs since fetching input data can be very expensive. 
```suggestion
        Task<IList<DurableOrchestrationStatus>> GetStatusAsync(IEnumerable<string> instanceID, bool showHistory = false, bool showHistoryOutput = false, bool showInput = false);
```